frontend/packages/vkt/src/utils/examCreateEvent.ts

```diff
@@ -16,40 +17,27 @@ export class ExamCreateEventUtils {
     );
   }
 
-  static isValidForm(examForm: DraftClerkExamEvent | undefined) {
-    if (!examForm) {
-      return false;
+  static isValidExamEvent(
+    examEvent: DraftClerkExamEvent | ClerkExamEvent | undefined
+  ) {
+    if (examEvent) {
+      return (
+        StringUtils.isNonBlankString(examEvent.language) &&
+        StringUtils.isNonBlankString(examEvent.level) &&
+        DateUtils.isValidDate(examEvent.date) &&
+        DateUtils.isValidDate(examEvent.registrationCloses) &&
+        DateUtils.isDatePartBefore(
+          examEvent.registrationCloses as Dayjs,
+          examEvent.date as Dayjs
+        ) &&
+        !ExamCreateEventUtils.maxParticipantsHasError(
+          true,
+          examEvent.maxParticipants
+        )
+      );
     }
```

Comment:
Tän vois lyhentää näin (ja poistaa `return false;` alhaalta):
```suggestion
    return (
      !!examEvent &&
      StringUtils.isNonBlankString(examEvent.language) &&
      StringUtils.isNonBlankString(examEvent.level) &&
      DateUtils.isValidDate(examEvent.date) &&
      DateUtils.isValidDate(examEvent.registrationCloses) &&
      DateUtils.isDatePartBefore(
        examEvent.registrationCloses as Dayjs,
        examEvent.date as Dayjs
      ) &&
      !ExamCreateEventUtils.maxParticipantsHasError(
        true,
        examEvent.maxParticipants
      )
    );
```